features/area-type-parents.private.feature

```diff
@@ -82,3 +82,55 @@ Background:
         """
 
         And the HTTP status code should be "404"
+
+    Scenario: Getting parent area count successfully
+
+        Given I am identified as "user@ons.gov.uk"
+
+        And I am authorised
+
+        Given the following parents areas count response is available from Cantabular:
+        """
+        {
+             "Dimension": {
```

Comment:
some heavy indenting there is that 4 space indenting?

Reply:
Yes it is.